scripts/ledger.lic

```diff
@@ -98,6 +106,25 @@ module Ledger
     end
   end
 
+  module Utility
+    def self.get_note_value(noteId)

```

Comment:
use snake_case_for_variables instead of CapilizationCasingOfVariables. That's the preferred ruby best practice.

Reply:
Damn, thought I had finally beat the habit...